docs/recipes.md

```diff
@@ -107,6 +107,37 @@ for path in sorted(Path("src").rglob("*.py")):  # (1)
 8. Actually write to the magic file.
 9. We can even set the `edit_uri` on the pages.
 
+> NOTE:
+> If your `edit_uri` is the default value or you have overwritten, you might
+> have to traverse the tree until you find the correct file from your `edit_uri`.
```

Comment:
```suggestion
> have to traverse the tree until you find the correct file for your `edit_uri`.
```

?

Reply:
Rewrote the paragraph to be more clear :)